saos-webapp/src/main/resources/message/common_pl.properties

```diff
@@ -133,6 +133,12 @@ judgmentSearch.results.paginationInfo = Pozycje od {0} do {1} spośród {2} wyni
 judgmentSearch.results.judgmentsNumber = {0,choice,0#0 orzeczeń|0<{0} orzeczenie|1<{0} orzeczenia|4<{0} orzeczeń} 
 judgmentSearch.results.link = Przejdź do strony szczegółów orzeczenia 
 
+judgmentSearch.hint.icon.alt = Ikona podpowiedzi - zaznaczenie spowoduje wyświetlenie tekstu podpowiedzi
+judgmentSearch.hint.reasons.title = Uzasadnienie podpowiedź
+judgmentSearch.hint.reasons.content = Typ uzasadnienie został stworzony, ponieważ dane z sądów powszechnych czasmi przychodza jako SENTENCE, a czasmi REASONS i nie da sie ch razem zmergować.
+judgmentSearch.hint.legalBases.title = Podstawy prawne podpowiedź
+judgmentSearch.hint.legalBases.content = Przeszukiwanie po polu podstawy prawne możliwe jest tylko w trybie tekstowym. Przykładowe wartości: art. 23 k.c. oraz art. 103 uksc.
```

Comment:
Podstawy prawne przypisane do orzeczeń w systemach źródłowych są przechowywane w ustandaryzowanym formacie, lecz jako zwykły tekst (np. 'art. 23 k.c. oraz art. 103 uksc.' lub 'art.32, 34 ustawy o zawodzie lekarza i lekarza dentysty'. To powoduje, że jest możliwe jedynie tekstowe przeszukiwanie po tym polu.